.circleci/config.yml

```diff
@@ -0,0 +1 @@
+version: 2
```

Comment:
We're not switching.
We had nothing before.
I'm adding this following your comment https://github.com/resin-io/resin-preload/pull/73#pullrequestreview-53270923 .